docs/getting-started/github-apps/github-apps.rst

```diff
@@ -59,11 +57,11 @@ With the Dockstore GitHub App installed, the synchronization is done automatical
 you add a new branch or release of a workflow on GitHub, Dockstore is notified,
 and Dockstore updates its copy of the workflow. For example, After publishing a new release
 of a workflow on GitHub, a new version of the workflow will be present in
-Dockstore shortly afterwards.
+Dockstore shortly afterwards. For this to work, a ``/.dockstore.yml`` file is **required in the root directory of each GitHub repository** you want
```

Comment:
Commented above and I know this makes it tricky, but we did add support for putting it in the .github directory.